backbone.layoutmanager.js

```diff
@@ -262,11 +262,6 @@ var LayoutManager = Backbone.View.extend({
     return this.__manager__.renderDeferred.promise();
   },
 
-  // Proxy `then` for easier invocation.
-  then: function() {
-    return this.promise().then.apply(this, arguments);
```

Comment:
I think the problem is with `resolveWith` since we are setting the resolved value to be the View. I don't think there's anything wrong with our thennable method, just our Promise resolution.